data/card_catalogue/upgradeable-1.json

```diff
@@ -201,7 +201,7 @@
     "details": {
       "rules": {
         "active": "Jump as high as you want.",
-        "passive": null
+        "passive": "Immune to damage from falling."
```

Comment:
Trying to make this a little more satisfying to hold for people who have upgraded up the chain